lib/agoneum/account/account.ex

```diff
@@ -63,7 +63,7 @@ defmodule Agoneum.Account do
   """
   def create_user(attrs \\ %{}) do
     %User{}
-    |> User.changeset(attrs)
+    |> User.registration_changeset(attrs)
     |> Repo.insert()
   end
```

Comment:
Should this file reside above the `account` directory? I thought it was conventional to have `foo.ex` live side-by-side with `/foo` but maybe I'm mistaken and/or that's changed.